### ISSUE TITLE
loading pre-training weights for training

### ISSUE THREAD

**Issue by GuuuuuG** [megvii-research/PETR#40]
Why is the loss still huge when loading pre-training weights for training, just like not loading them?

**Comment by yingfei1016** [comment]
Hi, can you provide config and run command?

**Comment by GuuuuuG** [comment]
<img width="414" alt="image" src="https://user-images.githubusercontent.com/110542663/184565263-b6e97f0a-e323-4bc6-a67e-651eecb4aff1.png">
<img width="762" alt="9043a0dc6252b7a348c6e32b64bdb0c6" src="https://user-images.githubusercontent.com/110542663/184565272-6d37a480-5ee0-483c-a03f-15ace15cea18.png">

**Comment by yingfei1016** [comment]
Hi，
You can send the config and log to my email: liuyingfei@megvii.com .

**Comment by GlennLoveZhen0124** [comment]
@yingfei1016  Hi, I got the same problem when finetuning, could you please provide some solution for this ?

**Comment by yingfei1016** [comment]
> got the same problem when finetuning

Hi，
Do you mean you have trained a model and then finetune it?
